rollbar/test/test_rollbar.py

```diff
@@ -523,7 +525,7 @@ def test_get_request_fastapi_router(self):
         def root(fastapi_request: Request):
             current_request = rollbar.get_request()
 
-            self.assertEqual(current_request, fastapi_request)
+            self.assertEqual(get_public_attrs(current_request), get_public_attrs(fastapi_request))
```

Comment:
Are the failsafe comments on this file below from GH Actions a concern?

Reply:
They come from the testsuite raising errors to ensure the library catches and reports errors correctly. One of these days I will make them go away, so our reports are cleaner. So, not a concern just an annoyance.